### PR TITLE
fix(textarea): update styles [khcp-6440]

### PR DESCRIPTION
**Pull request by aanchalm01**
# Summary
Addresses:
[https://konghq.atlassian.net/browse/KHCP-6440](https://konghq.atlassian.net/browse/KHCP-6440)

**Before:**
image 1093
image 1094
image 1095

**After:**
image 1096
image 1097
image 1098

## PR Checklist

* [ ] Does not introduce dependencies
* [ ] **Functional:** all changes do not break existing APIs and if so, bump major version.
* [ ] **Tests pass:** check the output of yarn test
* [ ] **Naming:** the files and the method and prop variables use the same naming conventions as other Kongponents
* [ ] **Framework style:** abides by the essential rules in Vue's style guide
* [ ] **Cleanliness:** does not have formatting issues, unused code (e.g., console.logs, debugger), or leftover comments
* [ ] **Docs:** includes a technically accurate README, uses JSDOC where appropriate

### CODE REVIEW

File: src/styles/forms/_inputs.scss [hunk 1]
```diff
@@ -131,16 +144,13 @@
...
-
-  &:focus {
-    box-shadow: inset 0 0 0 2px var(--KInputFocus, var(--blue-400)) !important;
-  }
-}
-
 .k-input-wrapper.input-error {
+  textarea.k-input.form-control {
+    box-shadow: none !important;
+    outline: 1.5px solid var(--red-500) !important;
```
Comment (adamdehaven) [inline src/styles/forms/_inputs.scss:150]: Need to use a while number for outline

File: src/styles/_mixins.scss [hunk 2]
```diff
@@ -33,6 +33,25 @@
...
+  outline: 1px solid var(--KInputHover, var(--blue-200)) !important;
+  transition: all 0.1s ease;
+}
+
+@mixin textarea-focus {
+  box-shadow: none !important;
+  outline: 1px solid var(--blue-400) !important;
+  transition: all 0.1s ease;
+}
+
```
Comment (portikM) [inline src/styles/_mixins.scss:54]: Thanks for moving this to mixins! 🙌

**Review by portikM — APPROVED** [review]
LGTM 🚀

File: src/styles/forms/_inputs.scss (hunk 1 above)
Comment (aanchalm01) [reply src/styles/forms/_inputs.scss]: Updated to 1px

**Review by portikM — APPROVED** [review]

**Pull request merged**